graylog2-server/src/main/java/org/graylog2/indexer/indices/stats/IndexStatistics.java

```diff
@@ -44,13 +45,18 @@ public static IndexStatistics create(String index,
         return new AutoValue_IndexStatistics(index, primaryShards, allShards, routing);
     }
 
+    //TODO: needs unit tests
     public static IndexStatistics create(String index, JsonNode indexStats) {
```

Comment:
should we deprecate this method to make sure we don't use it in the new client anymore? doesn't really feel right, but otherwise we might accidentally still use it instead of the new builder